Client/game_sa/CHudSA.cpp

```diff
@@ -635,8 +640,9 @@ void CHudSA::RenderHealthBar(int x, int y)
     float barWidth = useCustomSize ? componentProperties.hpBar.placement.customWidth : componentProperties.hpBar.placement.width;
 
     // Calc bar width depending on MAX_HEALTH stat
-    double statModifier = ((double(__cdecl*)(int))FUNC_CStats_GetFatAndMuscleModifier)(10);
-    float  totalWidth = (barWidth * maxHealth) / statModifier;
+    // We want to maintain the proportions of the bar and its width after changing MAX_HEALTH from 176 to 200
+    static const float baseWidth = barWidth * 100.0f / 176.0f; // 176 is default STAT_10 value
+    float totalWidth = baseWidth + (((barWidth - baseWidth) / 100.0f) * (maxHealth - 100.0f));
```

Comment:
```suggestion
    const float baseWidth = barWidth * 100.0f / 176.0f; // 176 is default STAT_10 value
    float totalWidth = baseWidth + (((barWidth - baseWidth) / 100.0f) * (maxHealth - 100.0f));
```
`static` is unnecessary here